src/app/components/PlayerPanel.ts

```diff
@@ -0,0 +1,132 @@
+import { Player } from '../../Player';
+import { Render } from '../utils/Render';
+import { View } from '../View';
+import { flatten } from 'lodash';
+import { AnimalNames } from '../../Enums/AnimalNamesEnum';
+
+export class PlayerPanel {
+  /**
+   * Creates PlayerPanel based on data given
+   * @param player accepts instance of current player
+   * @param view accepts instance of View componnet
+   */
+  constructor(private player: Player, private view: View) {}
+
+  createPlayerPanel(): HTMLElement {
+    return Render.elementFactory(
+      'div',
+      {
+        className: 'player-panel',
+        style: `background-color: ${this.player.theColor};`,
+      },
+      this.createPlayerBoard(),
+      this.createResultWindow(),
+      this.createButtonPanel(),
+    );
+  }
+
+  private createPlayerBoard(): HTMLElement {
+    return Render.elementFactory(
+      'div',
+      {
+        className: 'player-panel__board',
+      },
+      Render.elementFactory(
+        'div',
+        { className: 'player-panel__info' },
+        Render.elementFactory('img', {
+          src: this.player.theAvatar,
+          alt: `${this.player.theName}-avatar`,
```

Comment:
Very important! I think we did not pay attention to alt in other views. Should be checked